ddtrace/_trace/trace_handlers.py

```diff
@@ -106,14 +106,15 @@ def _get_parameters_for_new_span_directly_from_context(ctx: core.ExecutionContex
 
 
 def _start_span(ctx: core.ExecutionContext, call_trace: bool = True, **kwargs) -> "Span":
+    activate_distributed_headers = core.get_local_item("activate_distributed_headers")
```

Comment:
Do we need to use `core.get_local_item` instead of `ctx.get_local_item`? If so, we should leave a comment.

Reply:
No I'll switch it.